packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -1369,10 +1313,12 @@ export class IntervalCollection
 	 * {@inheritdoc IIntervalCollection.add}
 	 */
 	public add({
+		id,
```

Comment:
is this exposed just for testing? I see that it's not in the public API, but is there a good reason we need it? seems like you converted some tests to not rely on ids they specify at the start of the test. Main thing I can think of is having non-deterministic IDs is annoying when you're debugging tests, but for that I'd rather set up some way to mock uuid() to give friendly deterministic ids.

Reply:
yeah, makes sense. Even if someone were to abuse the existence of it with casting, it's not really an issue unless they also generate race condition IDs a la the problem we have with DDS ids and the reason we have data store aliasing.